examples/nlp/duplex_text_normalization/conf/duplex_tn_config.yaml

```diff
@@ -75,6 +75,11 @@ decoder_model:
   nemo_path: ${decoder_exp_manager.exp_dir}/decoder_model.nemo # exported .nemo path
   lang: ${lang}
 
+  # Options related to covering grammars for TN
+  use_cg: false # Use covering grammars to avoid catastrophic errors
+  neural_confidence_threshold: 0.99 # If the neural model is not confident, then use the covering grammars
+  n_tagged: 1
```

Comment:
please add a comment on what this arg means

Reply:
I added the comment.